src/patternfly/components/Nav/nav.scss

```diff
@@ -164,20 +164,46 @@
   --pf-c-nav__simple-list--nested__separator--MarginLeft: var(--pf-c-nav__simple-list-link--nested--PaddingLeft);
 
   // Modifier - current item
-  --pf-c-nav__horizontal-list-link--m-current--Color: var(--pf-global--active-color--300);
+  --pf-c-nav__horizontal-list-link--m-current--Color: var(--pf-global--link--Color);
   --pf-c-nav__horizontal-list-link--m-current--FontWeight: var(--pf-global--FontWeight--bold);
-  --pf-c-nav__horizontal-list-link--m-current--after--BackgroundColor: var(--pf-global--active-color--300);
+  --pf-c-nav__horizontal-list-link--m-current--after--BackgroundColor: var(--pf-global--link--Color);
 
   // Modifier - current item
   --pf-c-nav__tertiary-list-link--m-current--Color: var(--pf-global--link--Color);
   --pf-c-nav__tertiary-list-link--m-current--FontWeight: var(--pf-global--FontWeight--semi-bold);
   --pf-c-nav__tertiary-list-link--m-current--after--BackgroundColor: var(--pf-global--link--Color);
 
+  // Scroll Button
+  --pf-c-nav__scroll-button--Top: var(--pf-global--spacer--sm);
+  --pf-c-nav__scroll-button--ZIndex: var(--pf-global--ZIndex--md);
+  --pf-c-nav__scroll-button--Width: var(--pf-global--spacer--xl);
+  --pf-c-nav__scroll-button--Height: #{pf-size-prem(40px)};
+  --pf-c-nav__scroll-button--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-nav__scroll-button--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-nav__scroll-button--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-nav__scroll-button--hover--Color: var(--pf-global--link--Color);
+  --pf-c-nav__scroll-button--nth-of-type-1--BoxShadow: 20px 0 10px -4px rgba(255, 255, 255, .7); // This is a special one-off shadow for nav
+  --pf-c-nav__scroll-button--nth-of-type-2--BoxShadow: -20px 0 10px -4px rgba(255, 255, 255, .7); // This is a special one-off shadow for nav
+  --pf-c-nav__scroll-button--m-dark--nth-of-type-1--BoxShadow: 20px 0 10px -4px rgba(21, 21, 21, .7); // This is a special one-off shadow for nav
+  --pf-c-nav__scroll-button--m-dark--nth-of-type-2--BoxShadow: -20px 0 10px -4px rgba(21, 21, 21, .7); // This is a special one-off shadow for nav
+
   // Base styling
-  overflow-x: auto;
+  position: relative;
 
-  // Hide scrollbars
-  &::-webkit-scrollbar {
+  &.pf-m-dark {
```

Comment:
I wonder if instead of `pf-m-dark`, you should just style the button `box-shadow` in the horizontal nav? And can the color and background colors not inherit from `@include pf-t-dark;` in `.pf-c-nav__horizontal-list`?